modules/core/src/v2/wallet.ts

```diff
@@ -63,7 +63,7 @@ export interface Memo {
  * - recipients
  */
 export interface BuildConsolidationTransactionOptions extends PrebuildTransactionOptions {
-  fromAddresses?: string[];
+  consolidateAddresses?: string[];
```

Comment:
Was this interface incorrect before? Seems there is no accompanying change in the implementation.

Reply:
yup, it was wrong but working if you did the js right